sites/next.skeleton.dev/src/pages/figma/index.astro

```diff
@@ -5,16 +5,157 @@ import LayoutRoot from '@layouts/LayoutRoot.astro';
+						Unsure how your design will look with<span class="text-surface-950-50"> a custom theme?</span>
+					</p>
+				</div>
+			</div>
+		</div>
+
+		<div class="pb-16">
+			<h2 class="h2 opacity-60">You’re not alone. We know these frustrations, and we’re here to help.</h2>
+		</div>
+
+		<h2 class="h2">We streamline your workflow by bringing design, development and your customers into perfect harmony.</h2>
+
+		<div class="flex flex-col gap-24 w-full py-12">
+			<div class="flex flex-col md:flex-row gap-6 w-full">
+				<div class="basis-1/2">
+					<Image src={ImgKit} class="rounded-container border border-surface-200-800" alt="components" />
+				</div>
+				<div class="flex flex-col gap-4 basis-1/2">
+					<h5 class="h5">Align quickly with prebuilt Skeleton components.</h5>
+					<p class="text-surface-800-200">
+						All components from the Skeleton design system are prebuilt and ready to use in Figma, saving you time and ensuring consistency.
+					</p>
+				</div>
+			</div>
+
+			<div class="flex flex-col-reverse md:flex-row gap-6 w-full">
+				<div class="flex flex-col gap-2 basis-1/2">
+					<h5 class="h5">Ensure consistency with theme imports and customizations.</h5>
+					<p class="text-surface-800-200">
+						Import your custom Skeleton themes directly into Figma using our plugin. Switch between dark and light modes or preview your
+						design with your theme instantly.
+					</p>
+				</div>
+
+				<div class="basis-1/2">
+					<Image src={ImgPlugin} class="rounded-container border border-surface-200-800" alt="components" />
+				</div>
+			</div>
+
+			<div class="flex flex-col md:flex-row gap-6 w-full">
+				<div class="basis-1/2">
+					<Image src={ImgTutorial} class="rounded-container border border-surface-200-800" alt="components" />
+				</div>
+				<div class="flex flex-col gap-2 basis-1/2">
+					<h5 class="h5">Iterate faster with tutorials designed for every step of the design process.</h5>
+					<p class="text-surface-800-200">Learn how to:</p>
+					<ul class="list-inside list-disc space-y-2 text-surface-800-200">
+						<li>Customize themes for your brand.</li>
+						<li>Create wireframes for quick iteration and client feedback.</li>
+						<li>Design developer-ready assets to streamline handoffs.</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+
+		<div class="flex flex-col gap-6 py-12">
+			<blockquote class="blockquote text-3xl not-italic">
+				"...the Figma is very well put together and the most robust that I could find out of all the UI kits so our company chose to go with
+				Skeleton 🙂
+			</blockquote>
+
+			<div class="flex flex-col gap-12 pb-12">
+				<a class="text-surface-700-300">Rob, in Skeleton community</a>
+				<a class="btn btn-lg preset-filled-primary-500 flex items-center gap-2 whitespace-nowrap shadow-lg w-fit" href="#get-access">
+					Join over 4500+ people
+				</a>
+			</div>
+		</div>
+
+		<h2 class="h2 pb-8" id="get-access">Get Access</h2>
+
+		<FigmaBuyNow />
```

Comment:
We need some kind of contact information below the Buy Now component. If users are having trouble, we don't want our contributors or community members being forced to deal with this. They should have a direct link of communication with you guys.

This can take one of more forms here:

- A direct support email list (add an anchor with a `mailto:` URL)
- A company website with your contact information
- We can also setup a `#figma-ui-kit` channel under Skeleton Labs on Discord.

I'd recommend something like this:

> The Skeleton Figma UI Kit is created with ❤️ from [Micha](https://www.linkedin.com/in/micha-mail%C3%A4nder-548aba16a/), [Bohdan](https://www.linkedin.com/in/bohdan-karpliak-659a41162/), and [Skeleton Labs](https://www.skeletonlabs.co/). If you have questions about your purchase or need support, please contact us at  [email@email.com](mailto:email@email.com).

If you guys don't have an email handy to use, then we can use `admin@skeleton.dev` temporarily. But I'd prefer you guys managed this asap.

Reply:
I like the suggestion with the #figma-ui-kit so we have one source for the community, and it has a search function, making it easier to deal with repeating questions 👍

Reply:
Note the `#figma-kit-support` channel has been added on Discord, but will remain private until the Kit launches.